package.json

```diff
@@ -13,7 +13,7 @@
     "dev": "babel-node src",
     "lint": "eslint src tests",
     "pretest": "npm run -s build",
-    "postinstall": "node lib/lib/check-engine.js",
+    "postinstall": "babel-node check.js",
```

Comment:
Oh right. Forgot not all babel pkgs are normal deps.

Let's rewrite to ES5 and just use `node`.